Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2575,6 +2575,65 @@ def to_context(self):
             return ret_value
 
 
+class IndicatorsTimeline:
+    """
+    IndicatorsTimeline class - use to return Indicator Timeline object to CommandResults
+    :type indicators: ``list`` or ``dict``
+
+    :param indicators: expects a list of indicators or a list of dict of indicators.
+    :type category: ``str``
+
+    :param category: indicator category.
+    :type message: ``str``
+
+    :param message: indicator message.
+    :return: a list of indicators
+
+    :return: None
+    :rtype: ``None``
+    """
+    def __init__(self, indicators=None, category=None, message=None):
+        # type: (list, str, str) -> None
+        if indicators is None:
+            indicators = []
+        else:
+            indicators = [indicators] if isinstance(indicators, dict) else indicators
+
+        timelines = []
+        timeline = {}
+        for indicator in indicators:
+            if isinstance(indicator, dict):
+                if 'Value' in indicator.keys():
+                    timeline['Value'] = indicator['Value']
+
+                if 'Category' in indicator.keys():
+                    timeline['Category'] = indicator['Category']
+                elif category:
+                    timeline['Category'] = category
+                else:
+                    timeline['Category'] = 'Integration Update'
```

Comment:
We can also add a condition to check if this integration or automation, and update the category accordingly